File: SDKs/amazonaws.com/greengrass/2017-06-07/java/lib/src/main/java/openapisdk/models/shared/ConnectorDefinitionVersion.java
package openapisdk.models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;

/**
 * ConnectorDefinitionVersion
 * Information about the connector definition version, which is a container for connectors.
**/
public class ConnectorDefinitionVersion {
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("Connectors")
    public Connector[] connectors;
    public ConnectorDefinitionVersion withConnectors(Connector[] connectors) {
        this.connectors = connectors;
        return this;
    }
}

File: SDKs/amazonaws.com/greengrass/2017-06-07/python/src/sdk/models/shared/connectordefinitionversion.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils
from ..shared import connector as shared_connector


@dataclass_json
@dataclasses.dataclass
class ConnectorDefinitionVersion:
    r"""ConnectorDefinitionVersion
    Information about the connector definition version, which is a container for connectors.
    """
    
    connectors: Optional[list[shared_connector.Connector]] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('Connectors') }})
    
